Show Database Details

pasindugayashan opened the pull request:
Show Database Details,
Total database count.
Total count of tables by databases
Total count of procedures by databases
Total count of views by databases

pasindugayashan committed:
Show Database Details

Show Database Details,
  Total database count.
  Total count of tables by databases
  Total count of procedures by databases
  Total count of views by databases

pasindugayashan committed:
Merge branch 'pasindu/initialize' of https://github.com/madmadusanka/DatabaseHelper into pasindu/initialize

pasindugayashan committed:
Merge branch 'main' into pasindu/initialize

madmadusanka commented on DatabaseHelper/DatabaseHelper/frmLandingPage.vb:
```diff
@@ -63,18 +75,111 @@ Public Class frmLandingPage
+        Try
+            If connection.State = ConnectionState.Open Then
+                ' Query to retrieve the count of tables in the database
+                Dim query As String = $"USE [{databaseName}]; SELECT COUNT(*) AS ViewCount FROM INFORMATION_SCHEMA.VIEWS"
```
Add all the hard coded sql query to Common.SQLQueries   Constant class (Module)

pasindugayashan committed:
Show Database Details

Show Database Details,
Total database count.
Total count of tables by databases
Total count of procedures by databases
Total count of views by databases

pasindugayashan committed:
Merge branch 'pasindu/initialize' of https://github.com/madmadusanka/DatabaseHelper into pasindu/initialize

pasindugayashan committed:
Show Database Details,

Total database count.
Total count of tables by databases
Total count of procedures by databases
Total count of views by databases
